### PR TITLE
Add AFLW2000-3D Dataset

### PR DESCRIPTION
**Pull request by lambdaflow**
Added AFLW2000-3D Dataset

Its class name that subclassed `DatasetBuilder` is `Aflw2k3d` 

Generated dataset info available at Gist:
https://gist.github.com/lambdaflow/c96cb2591980af5a725d81a3ef4bc692

### CODE REVIEW

**Comment by lambdaflow** [comment]
> I signed it!

**Comment by SoumyadeepJana** [comment]
@lambdaflow You might consider removing the DL_EXTRACT RESULT attribute if it is not used in the test file to clean up extra lines

File: tensorflow_datasets/image/aflw2k3d.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            "landmarks_68_3d_xy_normalized": tfds.features.Tensor(shape=(68, 2), dtype=tf.float32),
+            "landmarks_68_3d_z": tfds.features.Tensor(shape=(68, 1), dtype=tf.float32),
+        }),
+        urls=["http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/main.htm"],
+        citation=_CITATION,
+    )
+
+  def _split_generators(self, dl_manager):
+    extracted_path = dl_manager.download_and_extract(
+        "http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/Database/AFLW2000-3D.zip"
```
Comment (us) [inline tensorflow_datasets/image/aflw2k3d.py]: delete empty lines

File: tensorflow_datasets/image/aflw2k3d.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    )
+    return [
+        tfds.core.SplitGenerator(
+            name="test",
```
Comment (us) [inline tensorflow_datasets/image/aflw2k3d.py]: You have just 1 split and `test` not a good name. You should use `tfds.Split.TRAIN`

File: tensorflow_datasets/image/aflw2k3d_test.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import aflw2k3d
+
+
+class Aflw2k3dTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = aflw2k3d.Aflw2k3d
+  SPLITS = {
+      'test': 2,
+  }
+  DL_EXTRACT_RESULT = ''
```
Comment (SoumyadeepJana) [inline tensorflow_datasets/image/aflw2k3d_test.py]: You might consider removing the DL_EXTRACT RESULT attribute if it is not used in the test file to clean up extra lines

File: tensorflow_datasets/image/aflw2k3d.py (hunk 2 above)
Comment (lambdaflow) [reply tensorflow_datasets/image/aflw2k3d.py]: Yes, this dataset is usually used for evaluation the performance of facial landmark detection models.  There is another dataset from 3DDFA that named as 300W-LP that is usually used for training.

Maybe it is better to change the name from 'test' to tdfs.Split.ALL ?

File: tensorflow_datasets/image/aflw2k3d.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""AFLW2000-3D Datasets
+
+http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/main.htm
```
Comment (ChanchalKumarMaji) [inline tensorflow_datasets/image/aflw2k3d.py]: You can remove this line because it is already mentioned in `_info`.

Comment (lambdaflow) [reply tensorflow_datasets/image/aflw2k3d.py]: > You can remove this line because it is already mentioned in `_info`.

removed, thanks for mention it.

**Comment by lambdaflow** [comment]
@us @Conchylicultor @cyfra 
resolved conflicts in image/__init__.py.

**Comment by us** [comment]
@Conchylicultor @cyfra being_merged.

**Pull request merged**